src/Raven.Studio/typescript/models/database/index/index.ts

```diff
@@ -235,10 +236,21 @@ class index {
                 return "Idle";
             }
 
-           
-
             return "Normal";
         });
+
+        this.mapReduceIndexInfoTooltip = ko.pureComputed(() => {
+            let infoTextHtml;
```

Comment:
![image](https://user-images.githubusercontent.com/1250431/71114142-6d110400-21cf-11ea-98e5-92b159e9abd3.png)
so please use let indexTextHtml  = "";

and always append.

Reply:
done